bugwarrior/docs/services/youtrack.rst

```diff
@@ -0,0 +1,103 @@
+YouTrack
+========
+
+You can import tasks from your YouTrack instance using
+the ``youtrack`` service name.
+
+Example Service
+---------------
+
+Here's an example of a YouTrack target::
+
+    [my_issue_tracker]
+    service = youtrack
+    youtrack.host = youtrack.example.com
+    youtrack.login = turing
+    youtrack.password = 3n1Gm@
+
+The above example is the minimum required to import issues from
+YouTrack.  You can also feel free to use any of the
+configuration options described in :ref:`common_configuration_options`
+or described in `Service Features`_ below.
+
+Service Features
+----------------
+
+Host Options
++++++++++++++++++++++
+
+The ``youtrack.host`` field is used to construct a URL for
+the YouTrack server. It defaults to a secure connection scheme (HTTPS)
+on the standard port (443).
+
+Connecting on a different port::
+
+    youtrack.port = 8443
+
+Allow server to use a self-signed SSL certificate::
+
+    youtrack.verify_ssl = False
```

Comment:
@ryneeverett I added a new commit to the PR standardizing documentation of the `verify_ssl` option across those three trackers.